modules/snippets/technology-preview.adoc

```diff
@@ -0,0 +1,12 @@
+// When including this file, ensure that {FeatureName} is set immediately before
```

Comment:
I don't think that this file is meant to be here under the modules dir, it should probably be deleted